### PR TITLE
fixes #2386 - permission failure on create_lookup_value

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/models/lookup_value.rb [hunk 1]
```diff
@@ -58,4 +58,25 @@ def validate_list
...
+    # We get called again with the operation being set to create
+    return true if operation == "edit" and new_record?
+    current = User.current
+    allowed = case match
+      when /^fqdn=(.*)/
+        # check if current fqdn is in our allowed list
+        Host.my_hosts.where(:name => $1).exists?
+      when /^hostgroup=(.*)/
+        # check if current hostgroup is in our allowed list
+        Hostgroup.my_groups.where(:name => $1).exists?
```
Comment (ohadlevy) [inline app/models/lookup_value.rb]: I think this needs to change based on the host_group_matchers_inheritance setting, and then search by name or by label.

File: app/models/lookup_value.rb [hunk 2]
```diff
@@ -58,4 +58,25 @@ def validate_list
     errors.add(:value, _("%{value} is not one of %{rules}") % { :value => value, :rules => lookup_key.validator_rule }) and return false unless lookup_key.validator_rule.split(LookupKey::KEY_DELM).map(&:strip).include?(value)
   end
 
+  private
+
+  def enforce_permissions operation
+    # We get called again with the operation being set to create
+    return true if operation == "edit" and new_record?
+    current = User.current
```
Comment (ohadlevy) [inline app/models/lookup_value.rb]: left over?

**Comment by isratrade** [comment]
@domcleal, red on Ruby 2.0 and mysql but it says "no failures"???

**Comment by domcleal** [comment]
@isratrade it didn't get as far as running the test suite, failed with some bizarre error, ignore it.  Guess you didn't mean to close the PR?

**Comment by isratrade** [comment]
@domcleal, thanks, didn't mean to close

**Comment by domcleal** [comment]
You just did it again :)

**Comment by ohadlevy** [comment]
merged as 6c492cc - thanks!

**Pull request closed without merging**